src/malloc/mallocng/glue.h

```diff
@@ -3,7 +3,7 @@
 
 #include <stdint.h>
 #include <sys/mman.h>
-#include <pthread.h>
+//#include <pthread.h>
```

Comment:
is this still needed?

Reply:
It does not seem necessary anymore, will uncomment it